src/commands/publish/upload_form/mod.rs

```diff
@@ -45,7 +48,7 @@ pub fn build_script_upload_form(target: &Target) -> Result<Form, failure::Error>
 
             let script_path = package.main()?;
 
-            let assets = ProjectAssets::new(script_path, Vec::new(), kv_namespaces)?;
+            let assets = ProjectAssets::new(script_path, Vec::new(), kv_namespaces, Vec::new())?;
```

Comment:
IT looks like if this vector is empty, it is never used--think we can make this an `Option<>` instead of an empty vector?

Reply:
cool